frameworks/Python/aiohttp/nginx-entrypoint.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+set -e
+CORES=$(nproc)
+echo "$CORES cores detected, starting $CORES aiohttp workers..."
+
+for i in $(seq 0 $((CORES-1))); do
+  PORT=$((8000 + i))
+  echo "Starting worker on port $PORT"
+  python3 -O -m app.app --port $PORT &
+done
+
+echo "Waiting for all workers to be ready..."
+for i in $(seq 0 $((CORES-1))); do
+  PORT=$((8000 + i))
+  until nc -z localhost $PORT; do
+    echo "Waiting for port $PORT..."
+    sleep 0.1
+  done
+done
+
+cat > /aiohttp/nginx.conf <<EOF
+worker_processes auto;
+
+events {
+    worker_connections 65535;
+}
+
+http {
+    keepalive_requests 10000000;
+
+    upstream aiohttp {
+        least_conn;
+EOF
+
+for i in $(seq 0 $((CORES-1))); do
+  echo "        server 127.0.0.1:$((8000 + i));" >> /aiohttp/nginx.conf
```

Comment:
It'd be a little more work, but we could create unix sockets for all of these.
Config would look more like:
```
server unix:/run/aiohttp/aiohttp-{i}.sock fail_timeout=0;
```

Reply:
Always it's better sockets, for performance !!

Reply:
yes, thanks for suggestion i've updated pr